packages/react-renderer-demo/src/app/pages/renderer/renderer-api.md

```diff
@@ -1,12 +1,58 @@
 import RouterLink from 'next/link';
 import Link from '@material-ui/core/Link';
-import Grid from '@material-ui/core/Grid'
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import AnnouncementIcon from '@material-ui/icons/Announcement'
+import orange from '@material-ui/core/colors/orange'
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
 
 import ListOfContents from '../../src/helpers/list-of-contents';
 
 <Grid container item>
 <Grid item xs={12} md={10}>
 
+
+# Schema
```

Comment:
Why it is moved above the renderer-api props?

Reply:
Because I think it was easily missed before.